### DISCUSSION TITLE
Exclude domains from cache

### DISCUSSION THREAD

**Discussion by c-michail** [mokeyish/smartdns-rs#371] [Ideas]
Hello,
It would be nice to have a list to exclude eg. dynamic domains from caching. (especially with low TTL) so it will never serve stale data. (and custom minimum/max ttl)

Thank you

**Comment by mokeyish** [comment]
Do you mean this:

https://github.com/mokeyish/smartdns-rs/blob/6c19d4adaf3404d5eb98f2d30176a37ef7bae5cb/src/config/domain_rule.rs#L22


here is the documentation:
<img width="1218" alt="image" src="https://github.com/user-attachments/assets/1e01eec7-5b05-4e53-9902-a69541cd77ac">

**Comment by c-michail** [comment]
Yes it seems that did the job! (The -no-cache option wasn't on the default config file but on the github page)

Thank you
